project/Doc.scala

```diff
@@ -50,19 +51,22 @@ object Scaladoc extends AutoPlugin {
       docs
     })
 
-  def scaladocOptions(ver: String, base: File, plugins: Seq[File]): List[String] = {
-    val urlString = GitHub.url(ver) + "/€{FILE_PATH_EXT}#L€{FILE_LINE}"
+  def scaladocOptions(scalaBinaryVersion: String, ver: String, base: File, plugins: Seq[File]): List[String] = {
+    val urlString = GitHub.url(ver) +
+                    // supported from Scala 2.12.9
+                    (if (scalaBinaryVersion != "2.11") "€{FILE_PATH_EXT}#L€{FILE_LINE}" else "€{FILE_PATH}.scala")
```

Comment:
I removed an extra `/` at the beginning I noticed in the generated docs. Will save 1 byte per documented class.